okhttp-clients/src/main/java/okhttp3/OpeningOkHttpClient.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * (c) Copyright 2023 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// CHECKSTYLE:OFF
+package okhttp3;
+// CHECKSTYLE:ON
+
+// Opens up the constructor, this is made public in later versions of OkHttp
+public class OpeningOkHttpClient extends OkHttpClient {
```

Comment:
I tried a couple ways of trying to have `RemotingOkHttpClient` masquerade as a regular `OkHttpClient` but without internal access to the constructor and a closed garden builder API I was not able to do so.